### PR TITLE
Add `morpheus65535/bazarr` service example

### PR DESCRIPTION
**Pull request by ikaruswill**
No swagger docs found but I grabbed the request/response from the developer console in Chrome

Example response:
```json
{
  "data": {
    "bazarr_config_directory": "/config", 
    "bazarr_directory": "/app/bazarr/bin", 
    "bazarr_version": "1.1.1", 
    "operating_system": "Linux-5.15.74-rockchip64-aarch64-with", 
    "package_version": "v1.1.1-ls167 by [linuxserver.io](https://linuxserver.io)", 
    "python_version": "3.9.5", 
    "radarr_version": "4.2.4.6635", 
    "sonarr_version": "3.0.9.1549", 
    "start_time": 1666981051.104633, 
    "timezone": "Asia/Singapore"
  }
}
```

### CODE REVIEW

**Comment by ikaruswill** [comment]
Ref: Issue #26

File: content/en/docs/Examples/services.md [hunk 1]
```diff
@@ -526,6 +526,30 @@ service:
...
+      type: github
+      url: morpheus65535/bazarr
+      url_commands:
+      - type: regex
+        regex: v([0-9.]+)$
+    deployed_version:
+      url: https://bazarr.example.io/api/system/status
+      headers:
+      - key: X-API-KEY
+        value: <API_KEY>
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:546]:
```suggestion
      url_commands:
        - type: regex
          regex: v([0-9.]+)$
    deployed_version:
      url: https://bazarr.example.io/api/system/status
      headers:
        - key: X-API-KEY
          value: <API_KEY>
```
Have indented list items like this on the others

**Review by JosephKav — CHANGES_REQUESTED** [review]

File: content/en/docs/Examples/services.md (hunk 1 above)
Comment (ikaruswill) [reply content/en/docs/Examples/services.md:546]: Sure, I'll add another commit to indent my previous changes the same way.

Comment (JosephKav) [reply content/en/docs/Examples/services.md:546]: Oh hadn't noticed I'd missed it on those XD. Thank you :)

**Pull request merged**